packages/@uppy/dashboard/src/components/Dashboard.jsx

```diff
@@ -138,6 +138,7 @@ export default function Dashboard (props) {
             <FileList
               // eslint-disable-next-line react/jsx-props-no-spreading
               {...props}
+              singleFile={singleFile}
```

Comment:
can't we use `itemsPerRow` instead of a new prop? If that's 1 it means the same.

Reply:
No, it's based on width, so it's `3` for single file on full-width Dashboard for me.